tools/pypsa/features.yaml

```diff
@@ -3,38 +3,59 @@
 #
 # SPDX-License-Identifier: CC-BY-4.0
 
-# yaml-language-server: $schema=https://raw.githubusercontent.com/open-energy-transition/openmod-features/6fd6237/schema/schema.yaml
+# yaml-language-server: $schema=https://raw.githubusercontent.com/open-energy-transition/openmod-features/000bf8a/schema/schema.yaml
 features:
   asset__candidates:
     converters:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/user-guide/components/links/
+      value: y
     electricity_transmission:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/user-guide/components/lines/
+      value: y
     endogenous:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/user-guide/components/generators/
+        - https://docs.pypsa.org/latest/user-guide/components/lines/
+        - https://docs.pypsa.org/latest/user-guide/components/links/
+        - https://docs.pypsa.org/latest/user-guide/components/storage-units/
+        - https://docs.pypsa.org/latest/user-guide/components/stores/
+        - https://docs.pypsa.org/latest/user-guide/components/transformers/
+      value: y
     exogenous:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/user-guide/components/generators/
+        - https://docs.pypsa.org/latest/user-guide/components/lines/
+        - https://docs.pypsa.org/latest/user-guide/components/links/
+        - https://docs.pypsa.org/latest/user-guide/components/storage-units/
+        - https://docs.pypsa.org/latest/user-guide/components/stores/
+        - https://docs.pypsa.org/latest/user-guide/components/transformers/
+      value: y
     fluid_or_gas_distribution:
       source: []
       value: n
```

Comment:
Can a link be used to represent this? I wasn't thinking here about tracking pressure levels, but just any kind of non-electricity spatial carrier transfer

Reply:
Yeah, this should be updated to better reflect that nuance of the feature with "transport model" links considered as a baseline feature that any spatially disaggregated model would have

Reply:
Spun out into an issue (#29 )